nbgrader/apps/dbapp.py

```diff
@@ -36,7 +36,6 @@ class DbBaseApp(NbGrader):
 
     def start(self):
         if sys.platform != 'win32':
-            lister = ExchangeList(coursedir=self.coursedir, parent=self)
```

Comment:
Why was 'from ..exchange import ExchangeList' left in the imports if this line was removed?

Reply:
An oversight on my part, I've removed all unused imports I could find that relates to the exchange package now.